README.md

```diff
@@ -51,66 +51,321 @@ directory to the web root of your server.
+For more information about how Decap CMS works with internationalized content, see the [internationalization support documentation](https://www.decapcms.org/docs/beta-features/#i18n-support).
+
+## Disabling Internationalization
+
+If you do not need internationalization support for your site, it can be disabled either for specific collections or
+for the entire site.
+
+### Disabling Internationalization for a Specific Collection
+
+If your site is translated into more than two languages but you aren't translating all your content, any empty
+collection will not be output by Eleventy. For example, if your site is configured to use `en-CA`, `fr-CA` and
+`de-DE` but you are not translating your posts into `de-DE`, you can leave the `/src/collections/posts/de-DE`
+directory empty and the German posts index page will not be built. However, if you want to disable _all_
+internationalization for a specific collection, you can do so by:
+
+1. Deleting all directories but the default locale in the `/src/collections/<collection>/` directory. For example, to
+   disable localization for posts, delete the `/src/collections/posts/fr-CA/` directory.
+2. Modify the [`src/admin/config.yml`](src/admin/config.yml) collection block's `i18n` and `folder` properties for the
+   relevant collection. For example, to disable localization for posts, make the following changes in the `posts`
+   block:
+
+   ```diff
+   - i18n: true
+   + i18n: false
+   - folder: "src/collections/posts/"
+   + folder: "src/collections/posts/en-CA/" # Look for posts to edit in the default locale directory.
+   ```
+
+   Optionally, you could remove all subdirectories of `src/collections/posts`, moving the `en-CA` (or other default
+   language) directory's contents (with the exception of the [`en-CA.11tydata.js`](src/collections/posts/en-CA/en-CA.11tydata.js)
+   file) up into `src/collections/posts`. If you do this, you'll also need to make the following changes to the
+   collection's directory data file, in this case [`src/collections/posts.11tydata.js`](src/collections/posts.11tydata.js):
+
+   ```diff
+   - const i18n = require("eleventy-plugin-i18n-gettext");
+   const { generatePermalink } = require("eleventy-plugin-fluid");
+   
+   module.exports = {
+       layout: "layouts/post",
+       eleventyComputed: {
+   +       lang: data => data.defaultLanguage,
+   -       langDir: data => data.supportedLanguages[data.locale].dir,
+   +       langDir: data => data.supportedLanguages[data.defaultLanguage].dir,
+           permalink: data => {
+   -           const locale = data.locale;
+   -           return generatePermalink(data, "posts", i18n._(locale, "posts"));
+   +           return generatePermalink(data, "posts");
+           }
+       }
+   };
+   ```
+
+3. Remove localization of the index pages for the collection. In the case of posts, the [configuration file](eleventy.config.js)
+   creates indexes of posts in each language:
```

Comment:
It seems as though the instructions for this section were written with the assumption that the optional method of change from step 2 was used.